zerver/openapi/zulip.yaml

```diff
@@ -13217,25 +13225,28 @@ paths:
                           client capability and access the `user_settings` object instead.
 
                           [capabilities]: /api/register-queue#parameter-client_capabilities
-                      default_view:
+                      web_home_view:
                         deprecated: true
                         type: string
                         description: |
                           Present if `update_display_settings` is present in `fetch_event_types`
                           and only for clients that did not include `user_settings_object` in
                           their [`client_capabilities`][capabilities] when registering the event queue.
 
-                          The [default view](/help/configure-home-view) in Zulip, represented
+                          The [home view](/help/configure-home-view) in Zulip, represented
                           as the URL suffix after `#` to be rendered when Zulip loads.
 
                           Currently supported values are `all_messages` and `recent_topics`.
 
                           See [PATCH /settings](/api/update-settings) for details on
                           the meaning of this setting.
 
-                          **Changes**: Deprecated in Zulip 5.0 (feature level 89). Clients
-                          connecting to newer servers should declare the `user_settings_object`
-                          client capability and access the `user_settings` object instead.
+                          **Changes**: New in Zulip 8.0 (feature level 219). Previously this was
+                          called `default_view`.
```

Comment:
I am not sure whether we add the changes entries for deprecated fields, but the field is actually updated so we might need this. Other solution could be we do not rename the deprecated field and keep the setting name same and only use the new name in `user_settings` object.

Reply:
I think it doesn't particularly matter, since clients aren't going to read this, so going with whatever's easiest makes sense.